### PR TITLE
Change source request flow

### PR DESCRIPTION
**Pull request by Namiii**
Implemented the new Request builder flow for Source request

### CODE REVIEW

File: src/main/java/co/omise/models/Source.java [hunk 1]
```diff
@@ -113,44 +121,59 @@ public void setTerminalId(String terminalId) {
...
         }
 
-        public Create currency(String currency) {
+        public CreateRequestBuilder currency(String currency) {
             this.currency = currency;
             return this;
         }
 
-        public Create type(SourceType type) {
+        public CreateRequestBuilder type(SourceType type) {
```
Comment (pitiphong-p) [inline src/main/java/co/omise/models/Source.java]: Should we add support for other types in this PR?

Comment (Namiii) [reply src/main/java/co/omise/models/Source.java]: What are the other source types?

Comment (nuxzero) [reply src/main/java/co/omise/models/Source.java]: @Namiii Do you will do the installments in this PR or a new PR?

Comment (Namiii) [reply src/main/java/co/omise/models/Source.java]: @pitiphong-p if everything else is good, shall we close this PR and I'll start working on the `installments` PR.

**Pull request merged**